packages/web-components/src/design-system-provider/index.ts

```diff
@@ -571,3 +571,5 @@ export class FluentDesignSystemProvider extends DesignSystemProvider
   })
   public neutralOutlineFocusDelta: number;
 }
+
+export { DesignSystem };
```

Comment:
On line 47 you'll see this element actually omits a few properties from the DesignSystem interface. You should probably create an interface that is that omission, implement the new interface into the `FluentDesignSystemProvider` and export the new interface.